requirements.txt

```diff
@@ -8,3 +8,4 @@ juju>=0.0.1,<1.0.0
 #git+https://github.com/juju-solutions/python-libjuju.git#egg=juju
 enforce==0.3.1
 ubuntui>=0.1.1,<1.0.0
+jujuplugins
```

Comment:
@johnsca I figured that there was no harm in leaving it in. Saves the trouble for when we fix the juju ssh bug and uncomment code.